Added Cost Reference Section (#214)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/style_guidelines/general-formatting.adoc

image:images/no.png[no] We will fix this issue in the 18.3.4 release next February.

image:images/yes.png[yes] It is anticipated that an upcoming release will include a fix for this issue.

[[references-to-cost]]
== References to cost

> Avoid all references to the costs and charges of Red Hat products. While the IBM style guide blocks the usage of `free`, these references should also be avoided since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.

Reviewer: Just a few suggestions/clarifications

```suggestion
Avoid all references to the costs and charges of Red Hat products. While the IBM style guide recommends against using the term "free", any references to cost should also be avoided in product documentation since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.
```

> .Example phrases

Reviewer: ```suggestion
.Example problematic phrase
```

> Avoid all references to the costs and charges of Red Hat products. Although the _IBM Style Guide_ recommends against using the term "free", also avoid any references to cost in product documentation because they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.

Reviewer 2: @EricPonvelle  Suggestion >> Any cost information is best referenced in marketing materials. wdyt?

Author: @Amrita42 I updated the PR with your suggestion. Thanks!

Reviewer 2: Thanks @EricPonvelle

> Avoid all references to the costs and charges of Red Hat products. While the IBM style guide recommends against using the term "free", any references to cost should also be avoided in product documentation since they can confuse users and cause legal concerns. Any cost information is best left to marketing materials.

Reviewer 3: The general entry looks really good but I'm adding these small suggested edits to make it more aligned with our IBM and writing style for these items:
- Not to use while to mean although
- Not to use since to mean because
- Avoid 'should',  passives, and be prescriptive
- minimalism -for the last sentence. It's debatable but I think it seems a bit like fluff. And in documentation guidelines maybe it's not applicable to discuss what belongs in marketing material?

```suggestion
Avoid all references to the costs and charges of Red Hat products. Although the _IBM Style Guide_ recommends against using the term "free", also avoid any references to cost in product documentation because they can confuse users and cause legal concerns.
```

> * "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to me the product is free when it isn't.

Reviewer 4: Would it be helpful to add an example about avoiding "free" as well as "cost"? Optionally link to IBM SG section on "free" as in original discussion.

Reviewer 5: Do we need to scope cost a bit more? Do we need to state monetary costs? Learning a product might cost time and resources.

Reviewer 5: What about "no cost" or "without charge". These two phrases are mentioned in the IBM SG, but should also be misleading?

Should we state that an IBM SG entry exists for "free", but the RH SSG supersedes the guidance on the use of this term and related terms?

Reviewer 5: Sounds good, @EricPonvelle . Thanks!

> * "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to me the product is free when it isn't.

Reviewer 4: ```suggestion
* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean the product is free when it isn't.
```

> * "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean the product is free when it isn't.

Reviewer 6: ```suggestion
* "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

> * "at no initial cost" - This phrase was removed from the documentation because while it implies there are further costs, it can also be construed to mean that the product is free when it is not.

Reviewer: I know what you're talking about here - that it was removed during peer review. But other readers might not. Suggest rewording a bit (and adding another comma)

```suggestion
* "at no initial cost" - This phrase should be avoided in documentation because, while it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

> * "at no initial cost" - This phrase should be avoided in documentation because, while it implies there are further costs, it can also be construed to mean that the product is free when it is not.

Reviewer 3: ```suggestion
* "at no initial cost" - Avoid this phrase in documentation because, although it implies there are further costs, it can also be construed to mean that the product is free when it is not.
```

suggestion here is for similar reasons listed in the last comment ^^
